plugins/modules/ndo_schema_template_deploy.py

```diff
@@ -134,8 +134,10 @@ def main():
         method = "POST"
         payload = dict(schemaId=schema_id, templateName=template)
         if state == "deploy":
+            mso.request("schemas/{id}/validate".format(id=schema_id), method="GET")
```

Comment:
Do you not need to check the return and error out?

Reply:
no it will error out in the request module, wrote tests for this also which checks the error